dcm4che-core/src/main/java/org/dcm4che3/data/BulkData.java

```diff
@@ -126,6 +132,15 @@ public String getURI() {
         return uri;
     }
 
+    public URI toURI() {
+        return URI.create(getURI());
+    }
+
+    public URI toFileURI() {
+        return URI.create(uriWithoutQuery());
+    }
+
```

Comment:
URIs are very important to how BulkData works, so I'm moving toward using typed URIs as much as possible rather than the internal string.  The internal string is used all over the place, so it could not be removed without some major structural changes.